.gitignore

```diff
@@ -13,3 +13,5 @@ phpcs.phar
 
 */development-only/*
 
+package-lock.json
```

Comment:
@mikeallbutt package.json and package-lock.json can't be ignored.

Reply:
Yes. I have fixed it now. I didn't realise that it was going to affect others. (I thought it was just my local development.)

Reply:
You'll need to commit the fix.

Reply:
> You'll need to commit the fix.

Fixed and pushed.